crates/vizia_core/src/views/virtual_list.rs

```diff
@@ -19,75 +19,120 @@ pub enum VirtualListEvent {
 }
 
 impl VirtualList {
-    pub fn new<V: View, L, T>(
+    pub fn new<V: View, L: Lens, T: 'static>(
         cx: &mut Context,
         list: L,
-        height: f32,
-        item: impl Fn(&mut Context, usize, Index<L, T>) -> Handle<V> + 'static,
+        item_height: f32,
+        item_content: impl Fn(&mut Context, usize, Index<L, T>) -> Handle<V> + Copy + 'static,
```

Comment:
The `Copy` constraint is due to having to move the content closure into the binding closure. I *think* this is fine, but if at some point we find this isn't fine then a way around it is to wrap it in `Arc` and clone the arc into the closure.